[Editted] SQL Commands:ORDER BY (#3347)
Codecademy/docs: content/sql/concepts/commands/terms/order-by/order-by.md

FROM table_name
ORDER BY column_name ASC | DESC;
```

> It's also possible to `ORDER BY` **multiple columns**: 
> > _by seperating with a comma_

Reviewer: ```suggestion

It's also possible to `ORDER BY` multiple columns by separating them with a comma.

```

```
SELECT column_name_1
FROM table_name
ORDER BY column_name_1 ASC | DESC, column_name_2 ASC | DESC;
```

> -  assuming **_column_name_1_** and **_column_name_2_** exist in **table_name**

Reviewer: ```suggestion

This assumes `column_name_1` and `column_name_2` exist in `table_name`.
```

> 

Reviewer: ```suggestion
```
Remove blank line.
